cmd/doctor.go

```diff
@@ -27,40 +28,11 @@ import (
 var CmdDoctor = cli.Command{
 	Name:        "doctor",
 	Usage:       "Diagnose and optionally fix problems",
-	Description: "A command to diagnose problems with the current Gitea instance according to the given configuration. Some problems can optionally be fixed by modifying the database or data storage.",
-	Action:      runDoctor,
```

Comment:
Should we still keep the default action to keep compatible? Otherwise we need to add a break label.

Reply:
I can't really see the point of keeping the old format - it's not been well-documented, it's kinda an odd format of a command and keeping it around will likely break other things.